lib/documents/schemas/international_development_funds.json

```diff
@@ -3,14 +3,15 @@
   "base_path": "/international-development-funding",
   "format_name": "International development funding",
   "name": "International development funding",
-  "description": "Find and apply for funds to run international development projects",
+  "description": "Grants published before XX September 2020 were published by the Department for International Development (DFID)
```

Comment:
Will `XX September 2020` be changed to an actual date in a subsequent PR?

Reply:
yes, there are to-do's on the trello card and this is not to merge until then

Reply:
Probably worth updating PR title to `[DO NOT MERGE]Updated DFID document types to FCDO`